common/src/app/common/pages/changes.cljc

```diff
@@ -216,8 +216,9 @@
       [:type [:= :del-typography]]
       [:id ::sm/uuid]]]]])
 
-
-
+(sm/def! ::changes
```

Comment:
In main.data.workspace.notifications/schema:handle-file-change there is a reference of `cpc/changes`, that is not defined. If I define it this way, it works, but I'm not sure that this is the best way.

Reply:
The fix is OK, is clearly a bug, it was defined probably initially and accidentally deleted.